src/front-end/src/components/MenteeForm.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+import { Form, Button, Input } from 'antd';
+import TimezonePicker from 'react-bootstrap-timezone-picker';
+import 'react-bootstrap-timezone-picker/dist/react-bootstrap-timezone-picker.min.css';
+import firebase from '../firebase-init';
+import MenteeCompletion from './MenteeCompletion';
+
+// const { RangePicker } = DatePicker;
+
+// const { Option } = Select;
+
+class MenteeForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isSubmitted: false,
+    };
+    this.onFinish = this.onFinish.bind(this);
+    this.onFinishFailed = this.onFinishFailed.bind(this);
+  }
+
+  onFinish(values) {
+    this.setState({ isSubmitted: true });
+
+    firebase
+      .auth()
+      .createUserWithEmailAndPassword(values.emailInput, values.password)
+      .catch((error) => {
+        console.log(error);
+      });
+  }
+
+  onFinishFailed(values) {
+    this.setState({ isSubmitted: false });

```

Comment:
This isn't needed.

Reply:
I used it because there is an error that 'this' has to be used here.
You can pull it and run without it.